src/test/java/codeu/model/store/persistence/PersistentDataStoreTest.java

```diff
@@ -59,7 +59,7 @@ public void testSaveAndLoadUsers() throws PersistentDataStoreException {
     User resultUserOne = resultUsers.get(0);
     Assert.assertEquals(idOne, resultUserOne.getId());
     Assert.assertEquals(nameOne, resultUserOne.getName());
-    Assert.assertEquals("password", resultUserOne.getPassword());
+    Assert.assertEquals("password", resultUserOne.getPassword()); //first param is plain text, second is hashed
```

Comment:
The comment isn't accurate here, since the password is the same as the plaintext one written above.